homeassistant/components/tautulli/__init__.py

```diff
@@ -63,5 +65,5 @@ def __init__(
             entry_type=DeviceEntryType.SERVICE,
             identifiers={(DOMAIN, user.user_id if user else entry_id)},
             manufacturer=DEFAULT_NAME,
-            name=user.username if user else DEFAULT_NAME,
+            name=user.username.capitalize() if user else DEFAULT_NAME,
```

Comment:
Why do we capitalize the user name?

Reply:
It is first in the name instead of Tautulli. Should Tautulli be added in front of the username or do we just not capitalize the username?

Reply:
I don't think it is correct to capitalize usernames in general.